commands/anonSuggest.js

```diff
@@ -0,0 +1,19 @@
+require('discord.js');
+
+module.exports = {
+	name: 'anonsuggest',
+	description: 'Posts an anonymous Suggestion',
+	usage: 'anonsuggest',
+	helpMsg: true,
+	execute(message, args) {
+		const {modBot} = message.client.config.channelIDs;
+
+		console.log('running');
+		const suggestion = args.join(' ');
+
+		message.client.channels.get(modBot).send('"' + suggestion + '": ' + message.author).then(async sent => {
```

Comment:
Put the `.then` on a new line, most other uses of it put it on a new line. Let's see if I can add this in linter too xD